### PR TITLE
[Term Entry] Kotlin `.ln()` method

### LINKED ISSUE #3526: [Term Entry] Kotlin Math .ln()
### Reason/inspiration (optional)

We would like a new term entry on the `.ln()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/ln/ln.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by manviii27**
### Description

A new term entry for Kotlin `.ln()` method.
Close #3526 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/ln/ln.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Syntax
+
+```pseudo
+ln(n)
+```
+
+- n: The number of type `double` for which the logarithm is to be calculated.
+
+The `.ln()` method returns a value of type `double`, the logarithm of `n` to `e`. The following are the special cases:
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/ln/ln.md:25]:
```suggestion
The `.ln()` method returns a value of type `double`, the logarithm of `n` to _e_. The following are the special cases:

```

File: content/kotlin/concepts/math-methods/terms/ln/ln.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+}
+```
+
+The output of this code will be:
+
+```shell
+1.791759469228055
+```
+
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/ln/ln.md:50]: Codebytes does not support Kotlin so this needs to be changed to a regular example or removed.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@manviii27 I did a first review and left comments. Most important is the codebyte must be changed or removed because codebytes does not support Kotlin.

**Comment by manviii27** [comment]
@SSwiniarski I've made the changes, please review!

**Review by SSwiniarski — COMMENTED** [review]
@manviii27 looks good for a second review.

**Review by SSwiniarski — APPROVED** [review]

File: content/kotlin/concepts/math-methods/terms/ln/ln.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.log()`** method in Kotlin's `math` class is used to calculate the natural logarithm of a number.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/ln/ln.md:15]:
```suggestion
The **`ln()`** method in Kotlin's `math` class is used to calculate the natural logarithm of a number.

```

File: content/kotlin/concepts/math-methods/terms/ln/ln.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Syntax
+
+```pseudo
+ln(n)
+```
+
+- n: The number of type `double` for which the logarithm is to be calculated.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/ln/ln.md:23]:
```suggestion
- `n`: The number of type `double` for which the logarithm is to be calculated.

```

File: content/kotlin/concepts/math-methods/terms/ln/ln.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+The `.ln()` method returns a value of type `double`, the logarithm of `n` to _e_. The following are the special cases:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/ln/ln.md:25]:
```suggestion
The `ln()` method returns a value of type `double`, the logarithm of `n` to _e_. The following values represent special cases:

```

File: content/kotlin/concepts/math-methods/terms/ln/ln.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+- `ln(NaN)` is `NaN`
+- `ln(n)` is `NaN` if `n < 0.0`
+- `ln(+Inf)` is `+Inf`
+- `ln(0.0)` is `-Inf`
+
+## Example
+
+This example shows how to use the `.ln()` method to calculate the natural logarithm of `6`:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/ln/ln.md:34]:
```suggestion
This example shows how to use the `ln()` method to calculate the natural logarithm of `6`:

```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@manviii27 I just added a few additional minor edits. Please update, and tag me when you've completed the changes. Thanks

**Comment by manviii27** [comment]
@caupolicandiaz I've made the changes, please review.

**Review by caupolicandiaz — APPROVED** [review]
@manviii27 This is ready for a second review

**Pull request merged**